gems.yml

```diff
@@ -111,7 +111,7 @@ gsl:
   description: Bindings for the GNU scientific library
   module: GSL
   owner: sciruby
-  version: "~> 2.1"
+  version: "~> 2.1.0.3"
 hamster:
```

Comment:
I don't think this makes any difference.

Reply:
May be it does because otherwise its not installing the version `2.1.0.3` which it should because its the latest. @v0dro Could you merge this?

Reply:
@v0dro ?